### PR TITLE
Added PAS support to OfflinePlaybackSampleApp

### PR DESCRIPTION
**Pull request by BCjwhisenant**
Testing:
- Verified that online streaming works with this configuration (a PAS-disabled account and a blank PAS token)
- Verified that online streaming works with a PAS-enabled account and valid token
- Verified that purchase is successful with this configuration (a PAS-disabled account and a blank PAS token)
- Verified that rental is successful with this configuration (a PAS-disabled account and a blank PAS token)
- Verified that purchase is successful with a PAS-enabled account and valid token
- Verified that rental is successful with a PAS-enabled account and valid token

### CODE REVIEW

File: brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java [hunk 1]
```diff
@@ -164,11 +171,15 @@ private void updateVideoList() {
             emptyListMessage.setText(R.string.fetching_playlist);
             emptyListMessage.setVisibility(View.VISIBLE);
 
-            playlist.findPlaylist(catalog, new PlaylistListener() {
+            HttpRequestConfig.Builder httpRequestConfigBuilder = new HttpRequestConfig.Builder();
+            httpRequestConfigBuilder.setBrightcoveAuthorizationToken(pasToken);
```
Comment (sergiomartinez4) [inline brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java]: are there any problems by passing a fake token? e.g. "YOUR_PAS_TOKEN"

File: brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java [hunk 2]
```diff
@@ -504,11 +518,18 @@ private void rentVideo(@NonNull final Video video) {
                 .setListener(new DatePickerFragment.Listener() {
                     @Override
                     public void onDateSelected(@NonNull Date expiryDate) {
-                        long playDuration = video.getDuration();
+                        // Set the playDuration value to the video duration plus 10000ms to account for:
+                        // - Loading the video into the player (which starts the playDuration clock)
+                        // - Starting playback in a manual-start player
+                        long playDuration = video.getDuration() + 10000;
```
Comment (sergiomartinez4) [inline brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java]: Please change `10000` with a static final constant

File: brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java (hunk 1 above)
Comment (BCjwhisenant) [reply brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java]: No. I've tested this change with an account not configured for PAS, and this token value is sent but ignored.

File: brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java (hunk 2 above)
Comment (BCjwhisenant) [reply brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java]: done

Comment (sergiomartinez4) [reply brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java]: thanks!

**Pull request merged**